beacon-chain/rpc/eth/rewards/structs.go

```diff
@@ -14,3 +14,29 @@ type BlockRewards struct {
 	ProposerSlashings string `json:"proposer_slashings"`
 	AttesterSlashings string `json:"attester_slashings"`
 }
+
+type AttestationRewardsResponse struct {
+	Data                AttestationRewards `json:"data"`
+	ExecutionOptimistic bool               `json:"execution_optimistic"`
+	Finalized           bool               `json:"finalized"`
+}
+
+type AttestationRewards struct {
+	IdealRewards []IdealAttestationReward
+	TotalRewards []TotalAttestationReward
```

Comment:
```suggestion
	IdealRewards []IdealAttestationReward `json:"ideal_rewards"`
	TotalRewards []TotalAttestationReward `json:"total_rewards"`
```

I think this should be in snake_case.